internal/command/plan.go

```diff
@@ -220,7 +220,8 @@ Plan Customization Options:
   -replace=resource   Force replacement of a particular resource instance using
                       its resource address. If the plan would've normally
                       produced an update or no-op action for this instance,
-                      Terraform will plan to replace it instead.
+                      Terraform will plan to replace it instead. You can use
+                      this option multiple times to include more than one object.
```

Comment:
```suggestion
                      this option multiple times to replace more than one object.
```

This wording is more appropriate for the `-replace` option, I think.

Reply:
Sounds better, thanks! :)